### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by AlexanderBerry**
Wrote dataReaderSNN.py to take the csv file as input and return a list of lists containing the training set, cross validation set, and test set.

### CODE REVIEW

**Comment by AlexanderBerry** [comment]
Merging to main branch
